tests/unit/dataactvalidator/test_fabsreq1_detached_award_financial_assistance.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'fabsreq1_detached_award_financial_assistance'
+
+
+def test_column_headers(database):
+    expected_subset = {'row_number', 'award_description', 'correction_delete_indicatr'}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ Test AwardDescription is required for all submissions except delete records. """
+
+    det_award = DetachedAwardFinancialAssistanceFactory(correction_delete_indicatr='C', award_description='Yes')
+    det_award_2 = DetachedAwardFinancialAssistanceFactory(correction_delete_indicatr='', award_description='No')
```

Comment:
Can we include a passing test where correction is `D` and award description is not None or ''? (applies to all the test_fabsreq passing tests)